userprefs.php

```diff
@@ -166,19 +166,28 @@ function do_font_sample_update(font_index, size_index, layout) {
     // font_sample when they change.
     window.addEventListener('DOMContentLoaded', () => {
         for (const orientation of ['v', 'h']) {
-            document.querySelectorAll('input[name=\"' + orientation + '_fntf\"]').forEach(radio => {
-                radio.addEventListener('change', function () {
-                    do_font_sample_update(this.value, null, orientation);
+            let radios = document.querySelectorAll('input[name=\"' + orientation + '_fntf\"]');
```

Comment:
querySelectorAll shouldn't return a falsy object, just an empty list per https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelectorAll. So we shouldn't need this check, right?

Reply:
You're right. I'll remove it.